src/Catalog.API/appsettings.json

```diff
@@ -17,6 +17,7 @@
   },
   "ConnectionStrings": {
     "EventBus": "amqp://localhost"
+    //"catalogOpenAi": "Endpoint=xxxx;Key=xxxx"
```

Comment:
One nice thing about Aspire is that we can centralize this configuration in the eShop.AppHost's configuration. Thus a local dev only needs to set this info in 1 place (either `eShop.AppHost\appsetting.json` or eShop.AppHost's User Secrets) and it will flow to both Catalog.API and to WebApp.

So can we move this to `eShop.AppHost\appsetting.json`?